go/vt/vtadmin/cluster/config.go

```diff
@@ -284,6 +288,17 @@ func (cfg Config) Merge(override Config) Config {
 	return merged
 }
 
+func formatID(id string) string {
+	if strings.Contains(id, "_") {
+		old_id := id
+		// gRPC can't process custom resolver names with underscores
+		id = strings.Replace(id, "_", "-", -1)
+		log.Infof("replaced underscores in old id '%s' with dashes to form new id '%s'", old_id, id)
```

Comment:
Isn't this very noisy if we log this each time?

Reply:
The clusters are (generally) only configured at start-up, or with dynamic clusters, on every request. It would get noisy on dynamic clusters,  so we can remove it.